packages/faceted-search/src/components/Badges/BadgeDate/BadgeDate.component.js

```diff
@@ -38,6 +38,7 @@ const BadgeDate = ({
 	readOnly,
 	removable,
 	displayType,
+	...rest
```

Comment:
can we pass only the required props ? like dataAttributes ?

Reply:
Well why not it's up to us i can do the change to restrict a bit